contracts/citycoin.clar

```diff
@@ -854,13 +854,9 @@
 )
 
 ;;;;;;;;;;;;;;;;;;;;; SIP 010 ;;;;;;;;;;;;;;;;;;;;;;
-;; name change: per https://github.com/stacksgov/sips/pull/25
-;; .sip-10-ft-standard.sip-010-trait
-(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-10-ft-standard.ft-trait)
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait.sip-010-trait)
```

Comment:
Can you add a comment with the testnet version here as well so its not forgotten / easier to find when deploying to testnet?

```lisp
;; testnet implementation
;; (impl-trait 'ST1HTBVD3JG9C05J7HBJTHGR0GGW7KXW28M5JS8QE.sip-010-trait.sip-010-trait)
```

Reply:
Done